### DISCUSSION TITLE
Split packages

### DISCUSSION THREAD

**Discussion by kpcyrd** [Homebrew/discussions#2209] [Writing Formulae/Casks] [answered]
hi!

I'm currently looking into creating a package for software that contains both a server and a client component. For [Arch Linux](https://github.com/archlinux/svntogit-community/blob/packages/rebuilderd/trunk/PKGBUILD) and [Alpine Linux](https://git.alpinelinux.org/aports/tree/testing/rebuilderd/APKBUILD) I've created a `rebuilderd` and `rebuilderd-tools` package, is something like this also possible with brew, or would I create a single package that contains both components?

Thanks!

**Comment by SMillerDev** [comment, marked as answer]
It's possible but generally they would be a single package in brew.

**Reply by kpcyrd** [reply]
Thanks for the quick reply, I'm going to create a single package then! :)
